docs/tutorials/whoami/README.mdx

```diff
@@ -502,6 +549,15 @@ The issuer URL can be found in your Keycloak Admin Console. In your 'mcp-realm',
 
 </TabItem>
 
+<TabItem value="asgardeo" label="Asgardeo">
+
+    You can find the issuer URL in the Asgardeo Console. Navigate to the created application, and open the **Info** tab. The **Issuer** field will be displayed there and should look like:
+    https://api.asgardeo.io/t/&lt;your-organization-name&gt;/oauth2/token
```

Comment:
Please use an inline code block for the URL.
Also, is the use of `&lt;` and `&gt;` intentional? Should it be `<your-organization-name>` instead?